ci/code_checks.sh

```diff
@@ -69,71 +69,6 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
     $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=EX01,EX02,EX04,GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,PR03,PR04,PR05,PR06,PR08,PR09,PR10,RT01,RT02,RT04,RT05,SA02,SA03,SA04,SS01,SS02,SS03,SS04,SS05,SS06
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
-    MSG='Partially validate docstrings (EX03)' ;  echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=EX03 --ignore_functions \
-        pandas.Series.plot.line \
```

Comment:
are you sure this can be removed? `ruff` only works statically, so only checks generated docstrings - any docstring with a substitution won't be checked. that's what `validate_docstrings.py` does

Reply:
Ah good point. Yeah none of the dynamic docstrings were able to get formatted so I'll revert this

Reply:
one more reason to get rid of dynamic docstrings 😉

Reply:
Big +1 to remove them!